socs/agents/scpi_psu/drivers.py

```diff
@@ -1,6 +1,161 @@
 # Tucker Elleflot
+
+import socket
+import time
+
 from socs.common.prologix_interface import PrologixInterface
 
+# append new model strings as needed
+ONE_CHANNEL_MODELS = ['2280S-60-3', '2280S-32-6', '9171',
+                      '9172', '9181', '9182', '9183', '9184', '9185']
+TWO_CHANNEL_MODELS = ['9173', '9174']
+THREE_CHANNEL_MODELS = ['2230G-30-1']
+
+# error codes from 2280S devices
+# part of an attempt to query devices for the number
+# of supported channels
+UNDEFINED_HEADER = -113
+HEADER_SUFFIX_OUT_OF_RANGE = -114
+
+
+class ScpiPsuInterface:
+    def __init__(self, ip_address, port, **kwargs):
+        self.ip_address = ip_address
+        self.port = port
+        self.sock = None
+        self.model = None
+        self.num_channels = 0
+        self.conn_socket()
+        try:
+            self.configure()
+        except ValueError as err:
+            raise ValueError(err)
+        super().__init__(**kwargs)
```

Comment:
This `super().__init__(**kwargs)` line can be safely removed, since `ScpiPsuInterface` doesn't inherit from a parent class. (This was probably copied from the [`PrologixInterface` class](https://github.com/simonsobs/socs/blob/99854a71a54347801eb1e8758219218856595458/socs/common/prologix_interface.py#L12) where it also isn't needed...)

Reply:
Almost forgot this one, but it's fixed now